go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	go.uber.org/mock v0.5.0
```

Comment:
It looks like bb-storage has not been bumped in this file. Please run a `go get -d -u ./...` or something to bump all of these.